.docs/content/1.installation/2.windows/3.useful-commands-for-wsl2.md

```diff
@@ -0,0 +1,46 @@
+# Useful commands for WSL2
+
+- To verify the WSL version:
+  ```powershell
+  wsl --list --verbose
+  ```
+
+- To view installed distro list:
+  ```powershell
+  wsl --list
+  ```
+
+- To change the default distro:
+  ```powershell
+  wsl --set-default <distro_name>
+  ```
+
+- To launch a distro:
+  ```powershell
+  wsl -d <distro_name>
+  ```
+
+- To launch the default distro:
+  ```powershell
+  wsl
+  ```
+
+- To launch a distro with a specific user:
+  ```powershell
+  wsl -d <distro_name> -u <user_name>
+  ```
+
+- To launch a command in a distro:
+  ```powershell
+  wsl -d <distro_name> -e <command>
+  ```
+
+- To lanch a distro in a specific directory:
+  ```powershell
+  wsl -d <distro_name> -w <directory>
+  ```
+
+- To uninstall a distro:
+  ```powershell
+  wsl --unregister <distro_name>
```

Comment:
This uninstalls a wsl instance/disk, not the distro itself (You can have multiple instances running Ubuntu with different names).

Reply:
That's not the meaning of MS documentation: https://learn.microsoft.com/en-us/windows/wsl/basic-commands#unregister-or-uninstall-a-linux-distribution

Reply:
Fair enough, sounds more like a semantic difference